app/helpers/vacancies_options_helper.rb

```diff
@@ -0,0 +1,42 @@
+module VacanciesOptionsHelper
+  def job_location_options
+    Vacancy::JOB_LOCATION_OPTIONS.map do |key, _value|
+      [I18n.t("helpers.fieldset.job_location_form.job_location_options.#{key}"), key]
+    end
+  end
+
+  def job_role_options
+    options = Vacancy::JOB_ROLE_OPTIONS.except(:nqt_suitable).map do |key, _value|
+      [I18n.t("jobs.job_role_options.#{key}"), key]
+    end
+    options.reject { |option| option == 'at_multiple_schools' } unless MultiSchoolJobsFeature.enabled?
```

Comment:
There should be a spec for this helper method asserting whether or not the option is included depending on if the feature is enabled or not

Reply:
Will add the test now